src/HealthChecks.CosmosDb/AzureCosmosDbHealthCheck.cs

```diff
@@ -31,23 +31,21 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
     {
         try
         {
+            if (_options is {DatabaseId: not null, ContainerIds: not null})
+            {
+                foreach (var containerId in _options.ContainerIds)
+                {
+                    var container = _cosmosClient.GetContainer(_options.DatabaseId, containerId);
```

Comment:
@leminh98 @Maya-Painter @sboshra @adityasa could someone from the CosmosDB Team please review this change?

Context: this code is executed quite frequently, from an ASP.NET health check which has almost 6m downloads on nuget.org: https://www.nuget.org/packages/AspNetCore.HealthChecks.CosmosDb

BTW we are soon going to implement https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/issues/2325 and hopefully reduce the number of users of Microsoft.Azure.DocumentDB.Core package